.github/workflows/ci-cd.yaml

```diff
@@ -17,17 +17,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node_version:
-          - 16.x
         os:
           - ubuntu-20.04
         experimental: [false]
     steps:
       - uses: actions/checkout@v2.4.0
-      - name: Node ${{ matrix.node_version }} - x64 on ${{ matrix.os }}
-        uses: actions/setup-node@v2.4.1
+      - name: Node x64 on ${{ matrix.os }}
```

Comment:
```suggestion
      - name: Node 16.x - x64 on Ubuntu 20.04
```

Reply:
- Remove strategy matrix lines 17-22 (until `steps`)
- Line 15 set `runs-on: ubuntu-20.04`
- Replace remaining (if any) references to `${{ matrix.* }}` variables